src/packages/@ncigdc/utils/analytics/performanceTracker.js

```diff
@@ -33,7 +28,21 @@ const performanceTracker = {
       label: eventFields[2],
       ...additionalProperties,
     };
+
     global.mixpanel.track(category, properties);
+
+    global.newrelic.addPageAction(label, {
+      ...properties,
+      name: label,
+    });
+
+    global.newrelic.addToTrace({
```

Comment:
Both work but seem to do different things

The addToTrace allows for this:
![image](https://user-images.githubusercontent.com/743976/27402390-79bfc7ee-5694-11e7-8534-71d2f28f74d8.png)


https://rpm.newrelic.com/accounts/1663638/browser/55008812/session_traces/177e2df8-e429-b747-ca74-015ccbcf6d48